.eslintrc.cjs

```diff
@@ -0,0 +1,114 @@
+const nodeFiles = [
+  '**/build.dflt.{js,mjs,cjs}',
+  '**/build.{js,mjs,cjs}',
+  '**/config/**',
+  '**/prebuild.{js,mjs,cjs}',
+  '**/prebuild/**',
+  '**/scripts/**',
+  'packages/new-design/lib/**',
+  'sites/backend/**',
```

Comment:
Note that when using a single centralized `.eslintrc`, we sometimes have to list specific directories where we want customized lint rules. An alternative would be to add new files, e.g. `sites/backend/.eslintrc`, that override parts (or all) of `./.eslintrc`.